classes/i18n/LocaleConversion.php

```diff
@@ -24,18 +24,41 @@
 namespace PKP\i18n;
 
 use Illuminate\Support\Arr;
+use PKP\core\Core;
 use PKP\facades\Locale;
 
 class LocaleConversion
 {
+    /**
+     * Get ISO639-2b array
+     */
+    protected static function getISO6392b(): array
+    {
+        $iso6392BFile = Core::getBaseDir() . '/' . PKP_LIB_PATH . '/lib/vendor/sokil/php-isocodes-db-i18n/databases/iso_639-2.json';
+        if (!file_exists($iso6392BFile)) {
+            return [];
+        }
+        return json_decode(file_get_contents($iso6392BFile), true);
```

Comment:
Given the file is static, it might be worth to cache this.

If you use the Laravel Cache, the underlying implementation will attempt to make use of the OPcache (if available on the server). If the data is small, it should be ok to cache it locally (`static $cache`).

ps: I'm considering this method can be called several times.

Reply:
OK, I am using Laravel Cache now...